web-apis/geolocation/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * A function to check if the client supports/allows geolocation API support
+ * @returns boolean True if the client allows, else false
+ */
+const isSupported = () => (navigator.geolocation ? true : false);
+
+/**
+ * A function that get's the coordinates of your current geo location
+ * and prints on the UI. If Geolocation API is not supported/allowed
+ * this function shows a valid error message.
+ */
+const geoFindMe = () => {
+  const status = document.querySelector('#error-text');
+  const mapLink = document.querySelector('#map-link');
+
+  mapLink.href = '';
+  mapLink.textContent = '';
+
+  const success = position => {
+      const latitude = position.coords.latitude,
+        longitude = position.coords.longitude;
```

Comment:
@wsameer Can you please show the accuracy information as well?

Reply:
@atapas Added accuracy information in the demo